app.py

```diff
@@ -42,7 +31,11 @@ def get(self):
 
     def post(self):
         content = request.json
-        validation.post_validation(content)
+        schema = RestaurantSchemaPost()
+        try:
+            schema.load(content)
```

Comment:
`Schema.validate()` can be used here instead of `load()`
https://marshmallow.readthedocs.io/en/stable/quickstart.html#validation-without-deserialization

Reply:
because `.load()` performs extra operations

Reply:
It doesn't throw Exception. Only returns errors like list

Reply:
did not know that, thanks